src/components/PageApplicationComments/PageApplicationComments.tsx

```diff
@@ -76,25 +77,33 @@ export const PageApplicationComments = ({
         <h1 className="govuk-heading-l">
           {type === "public" ? "Public Comments" : "Specialist Comments"}
         </h1>
-        <form
-          className="govuk-form"
-          method="get"
-          action={createPathFromParams(params, "comments/search")}
-          aria-label="Search & Sort comments"
-        >
-          <input type="hidden" name="type" value={type} />
-          <input type="hidden" name="council" value={councilSlug} />
-          <input type="hidden" name="reference" value={reference} />
-          <FormCommentsSearch searchParams={searchParams} />
-          <FormCommentsSort searchParams={searchParams} />
-        </form>
+        {pagination?.totalAvailableItems &&
+        pagination?.totalAvailableItems > 0 ? (
+          <form
+            className="govuk-form"
+            method="get"
+            action={createPathFromParams(params, "comments/search")}
+            aria-label="Search & Sort comments"
+          >
+            <input type="hidden" name="type" value={type} />
+            <input type="hidden" name="council" value={councilSlug} />
+            <input type="hidden" name="reference" value={reference} />
+            <FormCommentsSearch searchParams={searchParams} />
+            <FormCommentsSort searchParams={searchParams} />
+          </form>
+        ) : (
+          <></>
```

Comment:
You can simplify this from a ternary to `&&` to remove this empty fragment, `condition && <A />`